go.mod

```diff
@@ -386,4 +386,6 @@ replace (
 
 	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
 	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
+
+	github.com/smartcontractkit/chainlink-relay => github.com/smartcontractkit/chainlink-relay v0.1.7-0.20231107111643-de9168b2e6c5
```

Comment:
another way to do this simply `go get github.com/smartcontractkit/chainlink-relay@your_pr_commit` 

i find that easier than replacing. certainly, we must remove this replace and do go get before merging this

Reply:
Had as a replacement while waiting for the related PR to be merged. Will drop that before merging.